moment-msdate.js

```diff
@@ -27,11 +27,25 @@
 	}
 
 	const oaDateToTicks = function(oaDate) {
-		return ((oaDate - MS_DAY_OFFSET) * DAY_MILLISECONDS) + (oaDate >= 0.0 ? 0.5 : -0.5);
+		var ticks = ((oaDate - MS_DAY_OFFSET) * DAY_MILLISECONDS);
```

Comment:
Curious why the extra addition logic was removed during this update.  It looks like [Microsoft's implementation](https://github.com/Microsoft/referencesource/blob/3b1eaf5203992df69de44c783a3eda37d3d4cd10/mscorlib/system/datetime.cs#L554) has the addition logic...

Reply:
That part of the Microsoft implementation is a cast-to-long rounding/truncation hack, which doesn't make sense in JavaScript.  In fact, adding the extra 0.5 to the integer tick count causes round tripped values through Moment to be off by one millisecond in some cases.